services/venice-controller/src/main/java/com/linkedin/venice/controller/ZkHelixAdminClient.java

```diff
@@ -246,14 +246,13 @@ public void createVeniceStorageClusterResources(
       String clusterName,
       String kafkaTopic,
       int numberOfPartition,
-      int replicationFactor,
-      boolean isLeaderFollowerStateModel) {
+      int replicationFactor) {
     if (!helixAdmin.getResourcesInCluster(clusterName).contains(kafkaTopic)) {
       helixAdmin.addResource(
           clusterName,
           kafkaTopic,
           numberOfPartition,
-          isLeaderFollowerStateModel ? LeaderStandbySMD.name : VeniceStateModel.PARTITION_ONLINE_OFFLINE_STATE_MODEL,
```

Comment:
Maybe this doesn't come in scope of this PR, but we should remove `VeniceStateModel.PARTITION_ONLINE_OFFLINE_STATE_MODEL` too